templates/deployment.yaml

```diff
@@ -81,22 +88,61 @@ spec:
 {{ toYaml .Values.resources | indent 12 }}
         volumeMounts:
         - name: xwiki-data
+          mountPath: /usr/local/xwiki
+        - name: xwiki-solr-data
           mountPath: /opt/solr/server/solr
+        - name: xwiki-tomcat-conf
+          mountPath: /usr/local/tomcat/conf/web.xml
+          subPath: web.xml
       volumes:
+      - name: xwiki-solr-data
+        persistentVolumeClaim:
+          claimName: xwiki-solr
       - name: xwiki-data
         persistentVolumeClaim:
           claimName: xwiki
+      - name: xwiki-tomcat-conf
+        configMap:
+          name: xwiki-tomcat-conf
+          items:
+          - key: web.xml
+            path: web.xml
+
 ---
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: xwiki
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
 spec:
   accessModes:
-    - ReadWriteMany
+    - {{.Values.storageAccessMode}}
   resources:
     requests:
       storage: {{ .Values.storage }}
     {{- if .Values.storageClass }}
     storageClassName: {{ .Values.storageClass | quote }}
     {{- end }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: xwiki-solr
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
Think you want `include`.

For labels, I think the auto generated helm helpers file has a good example of auto-labeling.